components/nft-sale-widget/commons/sale-type-switch.tsx

```diff
@@ -24,13 +25,44 @@ export const SaleTypeSwitch = ({
           }
         }}
       />
+
       <Text
         fontFamily={"Inter Medium"}
         fontSize={{ base: 12, md: 14 }}
         color={"#585858"}
       >
         Original Sale
       </Text>
+      <Tooltip
+        label={
+          isOriginal ? (
+            <Text>
+              Toggle to view{" "}
+              <Show breakpoint="(max-width: 480px)">
+                <br />
+              </Show>{" "}
+              NFT copies
```

Comment:
As you can see only thing to change is the L44. and L52. No need to do entire text only partial text